pkg/monitor/monitor.go

```diff
@@ -241,9 +243,17 @@ func (m *Monitor) cleanupDriverNodePods(node *volume.NodeInfo) {
 	if err == nil {
 		return
 	}
-	pods, err := core.Instance().GetPods("", nil)
+
+	var pods *v1.PodList
+	if !reflect.ValueOf(storkcache.Instance()).IsNil() {
```

Comment:
nit: if storkcache.Instance() != nil should work as well

Reply:
done